lib/rules/function-url-quotes/index.js

```diff
@@ -19,8 +19,22 @@ const meta = {
 	url: 'https://stylelint.io/user-guide/rules/function-url-quotes',
 };
 
+/**
+ * @param {import('postcss').Declaration} decl
+ * @returns {number}
+ */
+function getDeclarationValueIndex(decl) {
```

Comment:
nit: I didn't see a `get` util for this. Have I missed something? Alternatively, should this go into `utils/`?

Reply:
[suggest] `declarationValueIndex` may be used here:

https://github.com/stylelint/stylelint/blob/b4fd2d50f588c5c4adb7f63a1091393b427f1a13/lib/utils/declarationValueIndex.js#L9